files/paths.go

```diff
@@ -77,12 +77,22 @@ func GetPathRelativeTo(path string, basePath string) (string, error) {
 		basePath = "."
```

Comment:
NIT: Is the comment above this function now out of date, or should it be updated to mention it evaluates symbolic links?

Reply:
Good point - fixed in ddb2085!